### PR TITLE
[8.0][FIX] l10n_eu_oss

### PR DESCRIPTION
**Pull request by RodrigoBM**
In v8 it is necessary to create account_tax_code to assign them to taxes. If account_tax_code are not assigned to the taxes, they will not be passed to the accounting entries.

### CODE REVIEW

**Comment by RodrigoBM** [comment]
@pedrobaeza

File: l10n_eu_oss/hooks.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+from openerp import api, SUPERUSER_ID, _
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def post_init_hook(cr, registry):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
```
Comment (pedrobaeza) [inline l10n_eu_oss/hooks.py]: This shouldn't be a hook, but a migration script.

File: l10n_eu_oss/hooks.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                account_tax_id.write({
+                    "tax_code_id": code_c.id,
+                    "tax_sign": 1,
+                    "ref_tax_code_id": code_c.id,
+                    "ref_tax_sign": -1,
+                })
+
+
+def _upgrade_tax_code(env, account_tax_id, type):
+    account_tax_code = env["account.tax.code"]
```
Comment (pedrobaeza) [inline l10n_eu_oss/hooks.py]: Reuse existing code instead of duplicating it.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: l10n_eu_oss/hooks.py (hunk 2 above)
Comment (RodrigoBM) [reply l10n_eu_oss/hooks.py]: Done

File: l10n_eu_oss/hooks.py (hunk 1 above)
Comment (RodrigoBM) [reply l10n_eu_oss/hooks.py]: Done, it's my first migrations, I hope it is correct. :)

**Comment by RodrigoBM** [comment]
Se cambia el nombre del commit

**Comment by pedrobaeza** [comment]
As I can't push to your branch, being an organization one, I have pushed directly to the branch with your commit 529e43650279098e1bde163618eaeca0d70960cc + an extra one for finishing the work: c1cdb1c2f05c39b39c86ede7d8aaa0fa5cf04322.

Thus, I close this.

**Pull request closed without merging**